[DO-NOT-MERGE] Building Bonus tweaks + new way of getting metal

Insanegame27 opened the pull request:
BONUSES:
Added food bonus of 5 to saddler to simulate more effective saddles and harnesses being used in farming. 
Added food bonus of 10 to Merchants Quarter. I found it a bit odd that merchants would stop trading food at a fairground.

PRODUCTION
Added metal production of 1 to fairground - This is a way for people far removed to gain access to metals, albeit in small quantities.
     - To balance this, it now produces 1 less of goods and wealth - Traders gotta trade.
Added metal production of 2 to Merchant's quarter
     - Produces 2 less of goods and 5 less of wealth

Insanegame27 committed:
Building Bonus tweaks + new way of getting metal

BONUSES:
Added food bonus of 5 to saddler to simulate more effective saddles and harnesses being used in farming. 
Added food bonus of 10 to Merchants Quarter. I found it a bit odd that merchants would stop trading food at a fairground.

PRODUCTION
Added metal production of 1 to fairground - This is a way for people far removed to gain access to metals, albeit in small quantities.
     - To balance this, it now produces 1 less of goods and wealth - Traders gotta trade.
Added metal production of 2 to Merchant's quarter
     - Produces 2 less of goods and 5 less of wealth

WeaverofBrokenThreads commented:
I am not 100% sure, but I don't think this will work as far as adding metal production- a settlement needs a base resource before it can use the provides or bonuses- ie, like building a mine in a region with no metal, it won't do anything.

Insanegame27 commented:
Would giving every settlement extremely low metal production? The mine is only buildable on hills and mountains, so it wouldn't be able to be abused too much.

Insanegame27 committed:
Update of Announcements

Announcements adjusted for Insanegame27's food and metal production patch

Insanegame27 committed:
More food production, Granary building

Gave the bowyer a small amount of food production as well as a small bonus to food production
Added granary building, which gives a bonus to food production through more efficient storage and distribution of food.You need the mill, carpenter and market built before you can get it, though.
Increased food bonus from fishery from 1 to 5.
University now gives food bonus - research of ways to increase yield.

Insanegame27 commented:
Added Granary building, more food across the board.

WeaverofBrokenThreads commented:
Don't believe you can do that either. Your best bet is to ask Tom.

Zanaras commented:
Adding food is something that you really have to look at the cause and effect of. More food can easily destabilize the economy, as can other resources. This is one of the main reasons I'm not a fan of weaver's pull request.

That said, when you add new things: buildings, items, whatever; make sure you update the language files for them so the name of the building gets rendered and described properly: Resources/translations/ 

About half of my edits usually so far have been me forgetting to add descriptions and such.

Insanegame27 committed:
Update LoadBuildingData.php

tvogt commented:
This PR needs work. I see what you want to do, but I don't think this is the right approach.

adding a base metal production would actually work. Wealth works this way, it has a base of 0 and only buildings can generate an initial production. So it would work for metal. I kind of like the idea of having a tiny sliver of it. However, this change would have to check that the mine does not suddenly become available and improve metal production. Did you check that?

5 and 10% bonuses are MASSIVE. And merchants wouldn't stop trading at the fair, all buildings add up. I would be game for a small bonus, 1% for the saddler and 2% for the Merchants Quarter. Remember, they add up.

Also what Zanaras said, if you add new buildings, you need to add the names descriptions, at least in english, to the language files, or they won't show up properly in-game.
